lib/diagnostic.js

```diff
@@ -8,38 +8,64 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min () {
-
+  let result = arguments[0]
+  for (let i = 0, min = arguments.length; i < min; i++) {
+    const current = arguments[i]
+    result = current < result ? current : result
+  }
+  return result
 }
-
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum () {
-
+const sum = function sum (a, b) {
+  return a + b
```

Comment:
Yes, but you want this function to be able to accept multiple arguments like the one above ^^